fendermint/fendermint/vm/interpreter/src/fvm/state/query.rs

```diff
@@ -7,13 +7,14 @@ use std::{cell::RefCell, sync::Arc};
 use anyhow::{anyhow, Context};
 
 use cid::Cid;
-use fendermint_vm_actor_interface::system::is_system_addr;
+use fendermint_vm_actor_interface::system::{is_system_addr, State, SYSTEM_ACTOR_ADDR};
```

Comment:
```suggestion
use fendermint_vm_actor_interface::system::{is_system_addr, State => SystemState, SYSTEM_ACTOR_ADDR};
```

I got a bit confused what `State` we were referring to.